radio/src/gui/9X/lcd.cpp

```diff
@@ -181,7 +181,7 @@ void lcd_putcAtt(coord_t x, coord_t y, const unsigned char c, LcdFlags flags)
     lcdPutPattern(x, y, q, 5, 6, flags);
   }
   else if (fontsize == TINSIZE) {
-    q = &font_3x5[((uint16_t)c-0x2d)*3];
+    q = &font_3x5[((uint16_t)c-0x20)*3];
```

Comment:
He based his branch on next instead of master so if it's fixed he wouldn't have noticed it... should rebase on master